doc/source/python-api.rst

```diff
@@ -144,6 +144,14 @@ Object Management
 .. autofunction:: tiledb.ls
 .. autofunction:: tiledb.walk
 
+Fragment Info
```

Comment:
Would you mind to add some small usage examples either here or inline (eg as doctests)? Later on we can move those to the gitbook docs...